modules/adding-a-custom-model-serving-runtime.adoc

```diff
@@ -25,7 +25,11 @@ NOTE: {productname-short} includes the OpenVINO Model Server model-serving runti
 . From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
 +
 The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled in your {productname-short} deployment. By default, the OpenVINO Model Server runtime is pre-installed and enabled in {productname-short}.
-. To add a new, custom runtime, click *Add serving runtime*.
+. To add a custom runtime:
+
+** To start with an existing runtime, click the More Options icon (image:images/osd-ellipsis.png[More options]) next to the existing rumtime and then click *Duplicate*. Skip to Step 5. 
```

Comment:
Rather than the icon, we use this throughout the RHOAI doc:

"click the action menu (&#8942;)"

Asciidoctor renders the Unicode sequence for the ellipsis icon nicely.

Reply:
Fix typo: "rumtime"